src/ngrx/5-dispatch-actions/contact.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
```

Comment:
Did we want to have a comment indicating the original file path in code files?

Reply:
This is a "fake" file :P